### PR TITLE
Use config installer instead of numiko_media

### PR DESCRIPTION
**Pull request by bboro**
This is working for me currently. 

I think we might want to fork the smg-websites repo and work on some fixes there. I suspect there are a lot of issues arising from outdated modules and core compatibility. I can't even `drush cr` at the moment fer example.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,10 @@ include .env
 
 default: up
 
-DRUPAL_ROOT ?= /var/www/html/web
+APP_ROOT ?= /var/www/html
+DRUPAL_ROOT ?= $(APP_ROOT)/docroot
+DRUPAL_SITE_DIR ?= $(APP_ROOT)/sites/default
```
Comment (andriokha) [inline Makefile]: We should kill this line - it's not doing anything at the mo and it's incorrect

Comment (bboro) [reply Makefile]: Hey! This should be fixed now to point to the correct place as w discussed :)

Comment (bboro) [reply Makefile]: Let's get this into the master and then we can migrate master into the bitbucket private repo (merge with site repo)

**Review by andriokha — APPROVED** [review]
Ta!

**Pull request merged**